database/schema.sql

```diff
@@ -0,0 +1,49 @@
+-- psql -U tyler -d Recommendation -a -f database/schema.sql
+DROP DATABASE IF EXISTS recommendation;
+CREATE DATABASE recommendation;
+
+CREATE TABLE listings (
+  listing_id INT NOT NULL PRIMARY KEY,
+  market VARCHAR(50),
+  neighbourhood VARCHAR(50),
+  room_type VARCHAR(50),
+  review_scores_rating INT
+);
+
+CREATE TABLE search_queries (
+  search_id INT NOT NULL PRIMARY KEY,
+  market VARCHAR(50),
+  searched_at DATE NOT NULL,
+  room_type VARCHAR(50),
+  max_price MONEY,
+  check_in DATE NOT NULL,
+  check_out DATE NOT NULL
+);
+
+CREATE TABLE search_results (
+  search_id INT NOT NULL REFERENCES search_queries(search_id),
+  listing_id INT NOT NULL REFERENCES listings(listing_id),
+  scoring_rules TEXT
+);
+
+CREATE TABLE booked_nights (
+  listing_id INT NOT NULL REFERENCES listings(listing_id),
+  booked_at DATE NOT NULL,
+  price MONEY,
+  search_id INT NOT NULL 
+-- // REFERENCES search_queries(search_id)
```

Comment:
Is your decision to not make search_id a foreign key constraint? Suggest removing commented out code before commit.

Reply:
Removed line. I am going to remove this because of the way data will be captured from the message bus.